analytics/population/nc_pop_recent.ipynb

```diff
@@ -0,0 +1,2597 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.    acs_demo['GEO_ID']= acs_demo['GEO_ID'].str.replace('1400000US','')
</pre><p>Please leave a comment explaining why this is needed.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Added a comment to explain this step. </p>